src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -1,39 +1,79 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { postMethod } from '../../api/api';
 import './NewCommentForm.scss';
 
-export const NewCommentForm: React.FC = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
+type Props = {
+  postId: number,
+};
 
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
+export const NewCommentForm: React.FC<Props> = ({ postId }) => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [body, setBody] = useState('');
 
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
+  const createComment = () => {
+    return postMethod('/comments', {
+      postId,
+      name,
+      email,
+      body,
+    });
+  };
 
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
+  return (
+    <form
+      className="NewCommentForm"
+      onSubmit={() => {
+        createComment();
+      }}
     >
-      Add a comment
-    </button>
-  </form>
-);
+      <div className="form-field">
+        <input
+          type="text"
+          name="name"
+          placeholder="Your name"
+          className="NewCommentForm__input"
+          required
+          value={name}
+          onChange={(e) => {
+            setName(e.currentTarget.value);
+          }}
+        />
+      </div>
+
+      <div className="form-field">
+        <input
+          type="text"
```

Comment:
```suggestion
          type="email"
```
Якщо не має власної валідації, можна використати хоча б браузерну, бо зараз замість емейлу я можу ввести все, що завгодно і в мене це вийде